### PR TITLE
Fumadocs v14

### PR DESCRIPTION
**Pull request by fuma-nama**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const metadata: Metadata = {
 
 export default function RootLayout({ children }: { children: ReactNode }) {
   return (
-    <html lang="zh-Hant-TW" className={noto.className} suppressHydrationWarning>
+    <html lang="zh-TW" className={noto.className} suppressHydrationWarning>
```
Comment (kane50613) [inline src/app/layout.tsx:41]: 應該要用 `zh-Hant-TW` 沒錯吧

https://www.injerry.com/blog_view/125

File: tailwind.config.js [hunk 2]
```diff
@@ -9,5 +9,5 @@ export default {
     "./content/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/**/*.{jsx,tsx,mdx}",
   ],
-  presets: [createPreset({ preset: "neutral", addGlobalColors: true })],
+  presets: [createPreset({ preset: 'neutral', addGlobalColors: true })],
```
Comment (kane50613) [inline tailwind.config.js:12]: 這裏 prettier 沒跑到嗎

File: src/app/layout.tsx (hunk 1 above)
Comment (fuma-nama) [reply src/app/layout.tsx:41]: biome says it is invalid though, it suggested TW instead

File: tailwind.config.js (hunk 2 above)
Comment (fuma-nama) [reply tailwind.config.js:12]: probably yes, and biome fixed it

File: src/app/layout.tsx (hunk 1 above)
Comment (fuma-nama) [reply src/app/layout.tsx:41]: looking up the specs, `zh-Hant` should be used, there's TW too but it's not `Hant`

**Review by kane50613 — APPROVED** [review]

**Pull request merged**